content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: "Exercise 3.0 - Using Ansible to Implement Security"
+workshops: ansible_tower
+workshop_weight: 30
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Exercise 3.0 - Using Ansible to Implement Security
+
```

Comment:
Might want to remove or reword this. This page does not run the STIG, only the 800-53. Maybe we can talk about the STIG or add it as more reference material at the end.